src/pivotal-ui/components/tables/tables.scss

```diff
@@ -47,66 +47,21 @@ Class                 | Description
   }
 }
 
-/*doc
----
-title: Rounded Tables
-name: table_rounded
-parent: table
----
-
-This is the cute rounding style on our tables
-
-```html_example
-<div class="table-rounded">
-  <table class="table table-data table-divided">
-    <thead>
-      <tr>
-        <th>Wombat Type</th>
-        <th>Wombat Cuteness</th>
-        <th>Wombat Surprise</th>
-      </tr>
-    </thead>
-    <tbody>
-      <tr>
-        <td>Cuddly</td>
-        <td>Immense</td>
-        <td>12</td>
-      </tr>
-      <tr>
-        <td>World's largest</td>
-        <td>Gargantuan</td>
-        <td>100</td>
-      </tr>
-      <tr>
-        <td>Baby</td>
-        <td>Infinite</td>
-        <td>10000</td>
-      </tr>
-    </tbody>
-  </table>
-</div>
-```
-*/
-
-.table-rounded {
- padding: 0 !important;
- border-radius: 10px;
- border: 1px solid $gray-8;
- overflow: hidden;
-
- > table, > .table {
-  margin: 0 !important;
-  border: 0 !important;
-
-  > tr:last-child, .tr:last-child {
-   border-bottom: none !important;
-  }
-  > tbody:last-child, .tbody:last-child {
-   > .tr:last-child, tr:last-child {
-    border-bottom: none !important;
-   }
+.panel.panel-basic-alt > .panel-body {
```

Comment:
This is super hard to talk about in text, please bear with me. ;)

Cross component interactions should be handled at the component level not at the CSS level because you don't want to need to say how every component behaves based on the parent context that it has (that approach doesn't scale well -- you'd end up with n X n exceptions where n is the number of components you have. It would also make it hard to have a "normal" non-modified component as a child). 

The code below is great (removing borders on the table) but we don't want to make that dependent on being inside a panel. e.g. the table should never have a wrapping border as we expect it to always be inside a panel. This also means the table will work with any panel (one with a simple border, one with a drop shadow, one jerry hasn't imagined yet).